### PR TITLE
fix #8532 feat(nimbus): check for absolute data before appending counts

### LINKED ISSUE #8532: IndexError: list index out of range
Sentry Issue: [EXPERIMENTER-PROD-GQ](https://mozilla.sentry.io/issues/3995399888/?referrer=github_integration)

```
IndexError: list index out of range
(1 additional frame(s) were not displayed)
...
  File "markus/main.py", line 515, in _timer_decorator
    return fun(*args, **kwargs)
  File "experimenter/jetstream/tasks.py", line 27, in fetch_experiment_data
    raise e
  File "experimenter/jetstream/tasks.py", line 21, in fetch_experiment_data
    experiment.results_data = get_experiment_data(experiment)
  File "experimenter/jetstream/client.py", line 212, in get_experiment_data
    data.append_conversion_count(primary_metrics_set)
  File "experimenter/jetstream/models.py", line 272, in append_conversion_count
    absolute_primary_metric_vals.all[0].count = conversion_count
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-3301)

### PR DESCRIPTION
**Pull request by mikewilli**
Because

- an error ingesting one metric's results prevents other results from being available

This commit

- checks for an intermittent issue in the data to preempt the error

### CODE REVIEW

**Review by jaredlockhart — CHANGES_REQUESTED** [review]
Can we add a test for this case?

**Comment by mikewilli** [comment]
Yea I realized it was failing coverage so moved it back to draft.

File: experimenter/experimenter/jetstream/models.py [hunk 1]
```diff
@@ -261,6 +261,12 @@ def append_conversion_count(self, primary_metrics_set):
                     primary_metric_data, BranchComparison.ABSOLUTE
                 )
 
+                if (
+                    absolute_primary_metric_vals.all is None
+                    or len(absolute_primary_metric_vals.all) == 0
+                ):
```
Comment (brennie) [inline experimenter/experimenter/jetstream/models.py:267]: `bool(None)` and `bool([])` both eval to `False`, so you can do `if not absolute_primary_metric_vals.all`

File: experimenter/experimenter/jetstream/models.py [hunk 2]
```diff
@@ -261,6 +261,9 @@ def append_conversion_count(self, primary_metrics_set):
                     primary_metric_data, BranchComparison.ABSOLUTE
                 )
 
+                if not absolute_primary_metric_vals.all:
+                    continue
+
```
Comment (yashikakhurana) [inline experimenter/experimenter/jetstream/models.py:266]: I am not sure by having this if and continue, how the below 2 statements will affect counts? may be I don't have enough context, just trying to understand 🤔

**Review by yashikakhurana — COMMENTED** [review]
Just a general question

**Pull request merged**